app/models/track.rb

```diff
@@ -47,7 +47,10 @@ def self.for_repo(repo)
   end
 
   def recache_num_exercises!
-    update_column(:num_exercises, exercises.where(status: %i[active beta]).count)
+    filtered_exercises = exercises.where(status: %i[active beta])
+    filtered_exercises = filtered_exercises.where(type: PracticeExercise.model_name.name) unless course?
```

Comment:
Maybe this would be simpler?

```suggestion
    filtered_exercises = filtered_exercises.where(type: PracticeExercise.to_s) unless course?
```

Reply:
It would! I'll update